jax/experimental/array_api/_data_type_functions.py

```diff
@@ -124,9 +129,22 @@ def _promote_types(t1, t2):
     raise ValueError("No promotion path for {t1} & {t2}")
 
 
-def astype(x, dtype, /, *, copy=True):
-  return jnp.array(x, dtype=dtype, copy=copy)
-
+def astype(x, dtype, /, *, copy: builtins.bool | None = True, device: xc.Device | Sharding | Any | None = None):
```

Comment:
Why `Any` for `device` here?

Reply:
Accidental vestige from a copy-and-paste 😅